pkgs/development/python-modules/troposphere/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, pythonOlder
+
+  # python dependencies
+, awacs
+, cfn-flip
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "troposphere";
+  version = "4.1.0";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-7SA8Eu8P/b47pDiOyV25gxx25E5nP0yw96CRpb2Y1BY=";
+  };
+
+  propagatedBuildInputs = [
+    awacs
+    cfn-flip
+    typing-extensions
```

Comment:
only required for Python < 3.8: https://github.com/cloudtools/troposphere/blob/4.1.0/setup.cfg#L34